synapse/rest/admin/background_updates.py

```diff
@@ -105,3 +110,68 @@ async def on_GET(self, request: SynapseRequest) -> Tuple[int, JsonDict]:
             }
 
         return 200, {"enabled": enabled, "current_updates": current_updates}
+
+
+class BackgroundUpdateStartJobRestServlet(RestServlet):
+    """Allows to start specific background updates"""
+
+    PATTERNS = admin_patterns("/background_updates/start_job")
+
+    def __init__(self, hs: "HomeServer"):
+        self._is_mine_id = hs.is_mine_id
+        self._auth = hs.get_auth()
+
+        self._store = hs.get_datastore()
+
+    async def on_POST(self, request: SynapseRequest) -> Tuple[int, JsonDict]:
+        requester = await self._auth.get_user_by_req(request)
+        await assert_user_is_admin(self._auth, requester.user)
+
+        body = parse_json_object_from_request(request)
+        assert_params_in_dict(body, ["job_name"])
+
+        job_name = body.get("job_name")
+
+        if job_name == "populate_stats_process_rooms":
+            await self._store.db_pool.simple_insert(
```

Comment:
I think these inserts will fail if the job is currently inserted, do we care about that or should we just let it error?

Reply:
Ok. I did not realized this. What is a good solution? Error `500` or status `200` with a message?

Reply:
Done.